src/roswire/proxy/roslaunch/reader.py

```diff
@@ -582,7 +619,7 @@ def locate_node_binary(self,
             path_in_scripts_dir = os.path.join(package_dir, 'scripts', node_type)
             path_in_nodes_dir = os.path.join(package_dir, 'nodes', node_type)
             if files.isfile(path_in_scripts_dir) and \
```

Comment:
Avoid breaking if conditions over multiple lines to improve readability. In this case, the simplest fix is probably to stick to a slightly longer line and add `# noqa: E501` to the end of it.

Reply:
Done